### PR TITLE
Refactoring to use the WEIGHT_CLASS defines.

### PR DESCRIPTION
**Pull request by SamHPurp**
No CL, this is purely backend stuff.

### CODE REVIEW

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -146,16 +146,18 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
...
+		if(WEIGHT_CLASS_NORMAL)
 			size = "normal-sized"
-		if(4.0)
+		if(WEIGHT_CLASS_BULKY)
 			size = "bulky"
-		if(5.0)
+		if(WEIGHT_CLASS_HUGE)
 			size = "huge"
+		if(WEIGHT_CLASS_GIGANTIC)
+			size = "gigantic"
```
Comment (SamHPurp) [inline code/game/objects/items.dm]: I couldn't see why this wasn't included @Fox-McCloud

File: code/game/objects/items/devices/megaphone.dm [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	desc = "A device used to project your voice. Loudly."
 	icon_state = "megaphone"
 	item_state = "radio"
-	w_class = 1
+	w_class = WEIGHT_CLASS_SMALL
```
Comment (Jovaniph) [inline code/game/objects/items/devices/megaphone.dm]: Wrong weight class?

Comment (SamHPurp) [inline code/game/objects/items/devices/megaphone.dm]: Welp thats true.

Although, why is a Megaphone classed as TINY in the first place... 

To keep this a re-factor I'll change this, but probably just be larger than TINY... as TINY is defined as "Usually items smaller then a human hand, ex: Playing Cards, Lighter, Scalpel, Coins/Money"

Comment (SamHPurp) [inline code/game/objects/items/devices/megaphone.dm]: Actually to simplify it, I'm going to remove it from this PR entirely and just make a separate one for it... Atomic commits and all that...

Comment (Jovaniph) [inline code/game/objects/items/devices/megaphone.dm]: Good idea

**Comment by SamHPurp** [comment]
This is ready to go when you're happy with it.

**Comment by SamHPurp** [comment]
This should be good to go. Could you check on the thing I commented on above regarding adding the Gigantic description. I couldn't see why it wasn't previously used. Happy to remove if you feel necessary, @Fox-McCloud

**Pull request merged**